core/ir/x86/opcode_api.h

```diff
@@ -1614,6 +1615,9 @@ enum {
     /* 1433 */ OP_vpopcntd, /**< IA-32/AMD64 vpopcntd opcode. */
```

Comment:
For the other 7 steps: 5 are here; OP_LAST looks like it is now auto-updating; decode_fast: looks like it should get the 3-byte size correct. 

Please add to the PR description that decode_fast needs no change.

Reply:
> decode_fast: looks like it should get the 3-byte size correct.


Okay thanks for checking. decode_fast looks complicated to update; glad that it's already good.